timbermill-java/timbermill-local/src/main/java/com/datorama/oss/timbermill/ElasticsearchClient.java

```diff
@@ -1101,8 +1112,9 @@ private void deleteIndex(String index) {
             } else {
                 LOG.info("Index {} deleted", index);
             }
-        } catch (IOException e) {
+        } catch (Exception e) {
```

Comment:
instead of rethrow, you can just not cache the exception here, and handle the catch in caller method (logging)

Reply:
i need to figure some thing about the delete
i will do it in a separate PR